### PR TITLE
Update sensors

### PR DESCRIPTION
**Pull request by kakopappa**
1. Remove power on or off from Sensors: Contact, Motion, Temperature, Air quality.  

2. Change Arduino Serial Baudrate to 115200

3. Fix LED light strip to support ESP32

4. Send event response check added

### CODE REVIEW

File: examples/AirQualitySensor/AirQualitySensor/AirQualitySensor.ino [hunk 1]
```diff
@@ -93,8 +92,13 @@ void loop() {
...
+  bool success = mySinricProAirQualitySensor.sendAirQualityEvent(pm1, pm2_5, pm10, "PERIODIC_POLL");
+  if(success) {
+    Serial.println("Air Quality event sent! ..");
+  } else {
+    Serial.printf("Something went wrong...could not send Event to server!\r\n");
+  }
+
    dispatchTime += MIN;
-   Serial.println("Sending Air Quality event ..");
  }
```
Comment (sivar2311) [inline examples/AirQualitySensor/AirQualitySensor/AirQualitySensor.ino:103]: Please correct the indent

File: examples/AirQualitySensor/AirQualitySensor_gp2y1014au0f/AirQualitySensor_gp2y1014au0f.ino [hunk 2]
```diff
@@ -91,34 +88,38 @@ void setupDustSensor() {
...
+  bool success = mySinricProAirQualitySensor.sendAirQualityEvent(pm1, pm2_5, pm10, "PERIODIC_POLL");
+  if(success) {
+    Serial.println("Air Quality event sent! ..");
+  } else {
+    Serial.printf("Something went wrong...could not send Event to server!\r\n");
+  }
+
+  dispatchTime += MIN;
+
+  Serial.println("Sending Air Quality event ..");
```
Comment (sivar2311) [inline examples/AirQualitySensor/AirQualitySensor_gp2y1014au0f/AirQualitySensor_gp2y1014au0f.ino:123]: Please correct the indent

File: examples/AirQualitySensor/AirQualitySensor/AirQualitySensor.ino [hunk 3]
```diff
@@ -93,8 +92,13 @@ void loop() {
    int pm2_5 = 0;   
    int pm10=0;   
    
-   mySinricProAirQualitySensor.sendAirQualityEvent(pm1, pm2_5, pm10, "PERIODIC_POLL");
+  bool success = mySinricProAirQualitySensor.sendAirQualityEvent(pm1, pm2_5, pm10, "PERIODIC_POLL");
+  if(success) {
+    Serial.println("Air Quality event sent! ..");
+  } else {
+    Serial.printf("Something went wrong...could not send Event to server!\r\n");
+  }
```
Comment (sivar2311) [inline examples/AirQualitySensor/AirQualitySensor/AirQualitySensor.ino:100]: Please correct the indent

File: examples/AirQualitySensor/AirQualitySensor_gp2y1014au0f/AirQualitySensor_gp2y1014au0f.ino (hunk 2 above)
Comment (kakopappa) [reply examples/AirQualitySensor/AirQualitySensor_gp2y1014au0f/AirQualitySensor_gp2y1014au0f.ino]: done

**Pull request merged**